pandas/core/generic.py

```diff
@@ -4232,7 +4232,8 @@ def as_matrix(self, columns=None):
 
     @property
     def values(self):
-        """Numpy representation of NDFrame
+        """
+        Return NDFrame as ndarray or ndarray-like depending on the dtype.
```

Comment:
I think https://github.com/pandas-dev/pandas/pull/20065 was working on `.values`. That has more comments so maybe remove your changes here.